flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing

Reply:
since it's built from source why limiting?

Reply:
I have no opportunity to test it on different architectures :-(

Reply:
ok, I removed the limitation to x86_64